### DISCUSSION TITLE
Why the admin functions are so lightweight?

### DISCUSSION THREAD

**Discussion by WestFarmer** [hoppscotch/hoppscotch#3582] [General]
This project is a very popular one and not young any more.
Why are backend management functions so lightweight?

What is the development team's main focus?

**Comment by liyasthomas** [comment]
Can you explain a bit more about the lightweight context here?

---

Hoppscotch strives to become a lightweight API development tool with only the essential features and components so that we can provide a bloatless interface for API development.

If you feel the currently available operations are not enough, feel free to request any specific features/improvements by creating a ticket from the [issues tab](https://github.com/hoppscotch/hoppscotch/issues/new/choose).
